tests/integration/aof.tcl

```diff
@@ -478,23 +478,21 @@ tags {"aof external:skip"} {
     }
 
     test {EVAL timeout with slow verbatim Lua script from AOF} {
-        create_aof $aof_dirpath $aof_file {
-            append_to_aof [formatCommand select 9]
-            append_to_aof [formatCommand eval {redis.call('set',KEYS[1],'y'); for i=1,1500000 do redis.call('ping') end return 'ok'} 1 x]
-        }
-
-        start_server [list overrides [list dir $server_path appendonly no lua-time-limit 1 aof-use-rdb-preamble no]] {
+        start_server [list overrides [list dir $server_path appendonly yes lua-time-limit 1 aof-use-rdb-preamble no]] {
             # generate a long running script that is propagated to the AOF as script
             # make sure that the script times out during loading
+            create_aof $aof_dirpath $aof_file {
+                append_to_aof [formatCommand select 9]
+                append_to_aof [formatCommand eval {redis.call('set',KEYS[1],'y'); for i=1,1500000 do redis.call('ping') end return 'ok'} 1 x]
+            }
             set rd [redis_deferring_client]
-            r config set appendonly yes
```

Comment:
so we use DEBUG LOADAOF on a server where `appendonly` is disabled.
isn't it relatively easy to just let it load that AOF file at startup rather than using debug (testing the more natural code path)?

Reply:
@oranagra I think you missed the `appendonly yes` in `start_server`, 
Maybe `$rd debug loadaof` is not necessary.

Reply:
ok, i did miss it. but still it may be better to start the server from that AOF file rather rather than load it later using DEBUG (more natural code path).
if there are complications, we can keep the current version though.

Reply:
Looking at the original test, we need to count the loading time, so we can't start directly from AOF.

Reply:
that timing isn't really necessary.. it's not part of the test, but rather just verbose printing.
p.s. now that we have #10058, we don't need it anymore anyway.

Reply:
@oranagra Have changed to let redis start from AOF sets. BTW i find a bug when many test use the same dir:
now in `start_server`, we use below to check whether the redis start is ready:
```
while 1 {
            # check that the server actually started and is ready for connections
            if {[count_message_lines $stdout "Ready to accept"] > 0} {
                break
            }
            after 10
        }
```
But this might be true right away because the stdout log was left over from the previous test.

So i changed to this:
```
set before_start_ready_count [count_message_lines $stdout "Ready to accept"]
spawn_server
while 1 {
          # check that the server actually started and is ready for connections
          if {[count_message_lines $stdout "Ready to accept"] == [expr $before_start_ready_count + 1]} {
              break
          }
          after 10
}

```

Reply:
maybe instead of counting twice, we can use `count_log_lines` and later `wait_for_log_messages`.
i.e. search in the log only starting a certain line.
but i see these functions use a server index and this code happens before `lappend ::servers $srv`, so that'll require more changes.
i guess i'm ok either way..

Reply:
I think calculation twice is fine, and it looks intuitive with minimal changes.